contracts/governance/Vesting/VestingRegistryLogic.sol

```diff
@@ -182,6 +183,15 @@ contract VestingRegistryLogic is VestingRegistryStorage {
                 uint256(VestingType.Vesting),
                 _vestingCreationType
             );
+
+        vestingDetail[vesting] = Vesting(
+            uint256(VestingType.Vesting),
+            _vestingCreationType,
+            vesting
+        );
+
+        emit VestingDetailPushed(vesting, vestingDetail[vesting]);
```

Comment:
```suggestion  
emit VestingDetailsSet(vesting, Vesting vestingDetails[vesting]);
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/79f72c867c17cd1f12ed1e726ee79e2c308a8e9e)